storage/storage.go

```diff
@@ -325,42 +335,108 @@ func (storage DBStorage) GetOrgIDByClusterID(cluster types.ClusterName) (types.O
 	return types.OrgID(orgID), nil
 }
 
+func parseRuleRows(rows *sql.Rows) ([]types.RuleOnReport, error) {
+	report := make([]types.RuleOnReport, 0)
+	for rows.Next() {
+		var details string
+		var ruleID string
+		err := rows.Scan(&details, &ruleID)
+		if err != nil {
+			log.Error().Err(err).Msg("ReportListForCluster")
+			return report, err
+		}
+		var ruleDetails json.RawMessage
```

Comment:
Why do you need to unmarshal to json.RawMessage and then convert to string when the DB field is already a string?